packages/components/src/composite/stories/index.story.tsx

```diff
@@ -230,7 +230,30 @@ This only affects the composite widget behavior. You still need to set \`dir="rt
 					'Composite.Group': commonArgTypes,
 					'Composite.GroupLabel': commonArgTypes,
 					'Composite.Row': commonArgTypes,
-					'Composite.Item': commonArgTypes,
+					'Composite.Item': {
+						...commonArgTypes,
+						accessibleWhenDisabled: {
```

Comment:
According to the Ariakit docs, the `accessibleWhenDisabled` prop is also available and supported for the `Composite` element. Should we add it there as well?

Reply:
Added, together with other props, as explained in https://github.com/WordPress/gutenberg/pull/63569#discussion_r1714041967